packages/core/src/index.ts

```diff
@@ -188,8 +187,6 @@ class Hopr extends EventEmitter {
       log('Addresses are sorted by default')
     }
     this.indexer = this.paymentChannels.indexer // TODO temporary
-
-    log(`using environment: ${this.environment.id}`)
```

Comment:
Why removing this?

Reply:
its a duplicate from an entry a couple of lines above